File: Code/dotNetBasics/ConsoleApp/ADStack/Components/ModelPredictiveControl.cs
using ConsoleApp.ADStack.Models;

/// <summary>
/// MPC
/// </summary>
internal class ModelPredictiveControl : IComponent
{
    public void Start()
    {
        Console.WriteLine("[ModelPredictiveControl] Starting.");
    }

    public void Stop()
    {
        Console.WriteLine("[ModelPredictiveControl] Stopping.");
    }
}

File: Code/Fluent.Java/src/adstack/components/ModelPredictiveControl.java
package adstack.components;

import adstack.models.IComponent;

public class ModelPredictiveControl implements IComponent {

	@Override
	public void start() {
        System.out.println("[ModelPredictiveControl] Starting.");
	}

	@Override
	public void stop() {
        System.out.println("[ModelPredictiveControl] Stopping.");
	}

}
